src/shared/util.js

```diff
@@ -294,6 +294,18 @@ const UNSUPPORTED_FEATURES = {
   smask: "smask",
   shadingPattern: "shadingPattern",
   font: "font",
```

Comment:
Assuming that the `font` key is actually unused now, should it be removed?

The same question also applies for `unknown` key above.

Reply:
These are exposed on the API, so I'll just mark them as `@deprecated` for now,

Reply:
> These are exposed on the API,

That's certainly true, but considering that nothing (in the PDF.js code) actually uses them now I just wondered what good they do for an API user at this point!? 

Anyway, it probably doesn't matter much either way here and the patch looks good as-is :-)